rome_app/templates/rome_templates/links.html

```diff
@@ -15,19 +15,32 @@
         background-size: contain;
         background-repeat:no-repeat;
     }
+
+    #links{
+        margin-top:5%;
+        margin-left:6.5%;
+        padding:5px;
+        font-weight: 200;
+        background:#F2D69E;
+        border-style:solid;
+        border-width:1px;
+        border-color:#D64833;
+    }
 </style>
 {% endblock %}
 
 {% block cookietrail %}
 {% endblock %}
 
 {% block content%}
-<h3>Links</h3>
+<div id="links">
+<h2>Links</h2>
```

Comment:
this is making the tests fail. Can you please update https://github.com/Brown-University-Library/ttwr/blob/master/unit_tests/test_views.py#L37, so the tests pass?